packages/shared/lib/services/on-event-scripts.service.ts

```diff
@@ -84,8 +84,15 @@ export const onEventScriptService = {
                 }
             }
             if (onEventInserts.length > 0) {
-                await trx.insert(onEventInserts).into(TABLE);
+                type R = Awaited<ReturnType<typeof onEventScriptService.update>>;
+                return await trx
+                    .insert<OnEventScript[]>(onEventInserts)
+                    .into(TABLE)
+                    .returning('*')
+                    .join('_nango_configs', `${TABLE}.config_id`, '_nango_configs.id')
+                    .select<R>([`${TABLE}.*`, '_nango_configs.provider_config_key as providerConfigKey']);
```

Comment:
must return the `providerConfigKey` and since it is not directly stored in the table but in the `nango_configs` one, I needed to add the join